lib/util/uid.js

```diff
@@ -0,0 +1,12 @@
+import nanoid from 'nanoid/generate';
+
+const CHARSET = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
```

Comment:
I'm using the custom charset in order to remove hyphens and underscores. We use those in some url slugs, so this helps simplify parsing.